docs/Dependency-injection.md

```diff
@@ -19,9 +19,11 @@ var mapper = new Mapper(Mapper.Configuration, childContainer.GetInstance);
 var dest = mapper.Map<Source, Destination>(new Source { Value = 15 });
 ```
 
-#### Gotchas
+#### QueryableExtensions
 
-Using DI is effectively mutually exclusive with using the IQueryable.ProjectTo extension method.  Use ``` IEnumerable.Select(_mapper.Map<DestinationType>).ToList() ``` instead.
```

Comment:
I think what this sentence is trying to say is that ProjectTo doesn't support DI, value resolvers and type converters, because it's just an expression to be interpreted by the EF provider. So you might want to make that more clear, not replace it.

Reply:
But that is already documented under QueryableExtensions, and has nothing to do with DI?

Reply:
That's not relevant. People make mistakes and this sentence is trying to prevent that.

Reply:
Added addional comment, with link to documentation.